lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -849,7 +851,39 @@ def test_student_progress_with_valid_and_invalid_id(self, default_store):
         # Assert that valid 'student_id' returns 200 status
         self.assertEqual(resp.status_code, 200)
 
-    def test_non_asci_grade_cutoffs(self):
+    def test_unenrolled_student_progress_for_credit_course(self):
+        """
+        Check if in credit course progress check for the student who is not enrolled in course raises None or not
+        """
+        # Create a new course, a user which will not be enrolled in course, admin user for staff access
+        course = CourseFactory.create()
```

Comment:
we should check this against both the split and mongo modulestores

(this is what the pattern looks like: https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/management/commands/tests/test_delete_orphans.py#L24-L31)